cmd/gotemplate/gotemplate_test.go

```diff
@@ -20,13 +20,21 @@ import (
 	"bytes"
 	"os"
 	"path"
+	"runtime"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func switchFileNotFoundError() string {
```

Comment:
We seem to be returning the same error message regardless of whether we are running on Windows or not. Should we just return the error message and remove the conditional?

Reply:
oops, my bad. The `else` was to be `open no-such-file.txt: no such file or directory` (message as given in unix systems); fixing.